src/sentry/static/sentry/app/components/smartSearchBar.jsx

```diff
@@ -71,7 +82,17 @@ class SmartSearchBar extends React.Component {
     /**
      * If this is defined, attempt to save search term scoped to the user and the current org
      */
-    recentSearchType: PropTypes.number,
+    savedSearchType: PropTypes.number,
+
+    /**
+     * Has pinned search feature
+     */
+    hasPinnedSearch: PropTypes.bool,
+
+    /**
+     * The pinned search object
+     */
+    pinnedSearch: PropTypes.object,
```

Comment:
Should we have a SentryType for saved search objects?

Reply:
Added